### PR TITLE
Fix proxy badge

### PR DESCRIPTION
**Pull request by ruihildt**

### CODE REVIEW

File: src/background/main.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { addExtensionsListeners } from '@/helpers/extensions';
-import { cleanProxyListeners, initProxyListeners } from '@/helpers/socksProxy';
+import {} from '@/helpers/socksProxy';
```
Comment (hankolsen) [inline src/background/main.ts:2]: You're not importing anything from socksProxy..?

File: package.json [hunk 2]
```diff
@@ -21,8 +21,9 @@
     "pack:zip": "rimraf mullvad-browser-extension-$npm_package_version.zip && jszip-cli add extension -o ./mullvad-browser-extension-$npm_package_version.zip",
     "pack:xpi": "cross-env WEB_EXT_ARTIFACTS_DIR=./ web-ext build --source-dir ./extension --filename mullvad-browser-extension-$npm_package_version.xpi --overwrite-dest",
     "start:chromium": "web-ext run --source-dir ./extension --target=chromium",
-    "start": "web-ext run --source-dir ./extension --target=firefox-desktop --start-url=about:debugging#/runtime/this-firefox",
-    "restart": "web-ext run --firefox-profile ~/.mozilla/firefox/md0hl4g1.MBE-Testing-Profile --keep-profile-changes --source-dir ./extension --target=firefox-desktop --start-url=about:debugging#/runtime/this-firefox",
+    "start": "web-ext run --source-dir ./extension --target=firefox-desktop --start-url=about:debugging#/runtime/this-firefox --start-url=mullvad.net/check",
```
Comment (hankolsen) [inline package.json:24]: Is it possible to use multiple `--start-url`?

**Review by hankolsen — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (ruihildt) [reply package.json:24]: Yes, multiple tabs will open.

File: src/background/main.ts (hunk 1 above)
Comment (ruihildt) [reply src/background/main.ts]: Fixed.

**Pull request merged**